src/contracts/core/DelegationManager.sol

```diff
@@ -311,117 +331,106 @@ contract DelegationManager is
      * @param strategy The strategy in which to increase the delegated shares.
-     * @param shares The number of shares to increase.
+     * @param existingPrincipalShares The number of deposit shares the staker already has in the strategy. This is the shares amount stored in the
+     * StrategyManager/EigenPodManager for the staker's shares.
+     * @param addedShares The number of shares to added to the staker's shares in the strategy
      *
-     * @dev *If the staker is actively delegated*, then increases the `staker`'s delegated shares in `strategy` by `shares`. Otherwise does nothing.
+     * @dev *If the staker is actively delegated*, then increases the `staker`'s delegated delegatedShares in `strategy`.
+     * Otherwise does nothing.
      * @dev Callable only by the StrategyManager or EigenPodManager.
      */
     function increaseDelegatedShares(
         address staker,
         IStrategy strategy,
-        uint256 shares
+        uint256 existingPrincipalShares,
+        uint256 addedShares
     ) external onlyStrategyManagerOrEigenPodManager {
         // if the staker is delegated to an operator
         if (isDelegated(staker)) {
             address operator = delegatedTo[staker];
+            uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategy);
 
-            // add strategy shares to delegate's shares
-            _increaseOperatorShares({operator: operator, staker: staker, strategy: strategy, shares: shares});
+            // add deposit shares to operator's stake shares and update the staker's depositScalingFactor
+            _increaseDelegation({
+                operator: operator,
+                staker: staker,
+                strategy: strategy,
+                existingPrincipalShares: existingPrincipalShares,
+                addedShares: addedShares,
+                totalMagnitude: totalMagnitude
+            });
         }
     }
 
+    // IGNORE THIS FUNCTION
     /**
-     * @notice Decreases a staker's delegated share balance in a strategy.
-     * @param staker The address to increase the delegated shares for their operator.
-     * @param strategy The strategy in which to decrease the delegated shares.
-     * @param shares The number of shares to decrease.
+     * @notice Decreases a staker's delegated delegatedShares for a strategy.
+     * @param staker The address to increase the delegated scaled shares for their operator.
+     * @param strategy The strategy in which to decrease the delegated scaled shares.
+     * @param removedShares The number of shares to decremented for the strategy in the
+     * StrategyManager/EigenPodManager
      *
-     * @dev *If the staker is actively delegated*, then decreases the `staker`'s delegated shares in `strategy` by `shares`. Otherwise does nothing.
+     * @dev *If the staker is actively delegated*, then decreases the `staker`'s delegated delegatedShares in `strategy` by `scaledShares`. Otherwise does nothing.
```

Comment:
there's a reference to `scaledShares` here which seems outdated, and I think there's a word missing in ` The number of shares to decremented`(?) -- is this supposed to be `to **be** decremented`, perhaps?

Reply:
Has since been fixed